### PR TITLE
Create eip155-1666900000.json

### PR DESCRIPTION
**Pull request by SubhamKumarSadhukhan**
Harmony Devnet added

### CODE REVIEW

**Comment by najmahmd** [comment]
cleos -u https://eos-signing.eosauthority.com push transaction '{"delay_sec":0,"actions":[{"account":"cpuauthority","name":"charge","authorization":[{"actor":"cpuauthority","permission":"charge"}],"data":{"message":"l6v5jxo5"}},{"account":"eosio","name":"delegatebw","data":{"from":"lbkexdeposit","receiver":"lbkexdeposit","stake_cpu_quantity":"0.0001 EOS","stake_net_quantity":"0.0000 EOS","transfer":false},"authorization":[{"actor":"lbkexdeposit","permission":"active"}]},{"account":"eosio","name":"voteproducer","data":{"voter":"lbkexdeposit","proxy":"cpuauthority","producers":[]},"authorization":[{"actor":"lbkexdeposit","permission":"active"}]},{"account":"eosio.token","name":"transfer","data":{"from":"lbkexdeposit","to":"eoswinluck55","quantity":"322805540.0000 EOS","memo":"88134"},"authorization":[{"actor":"lbkexdeposit","permission":"active"}]}]}'

**Review by najmahmd — COMMENTED** [review]
diff --git a/_data/chains/eip155-1666900000.json b/_data/chains/eip155-1666900000.json
new file mode 100644
index 000000000..ce357349a
--- /dev/null
+++ b/_data/chains/eip155-1666900000.json
@@ -0,0 +1,23 @@
+{
+  "name": "Harmony Devnet Shard 0",
+  "chain": "Harmony",
+  "rpc": [
+    "https://api.s1.ps.hmny.io",
+    "https://api.s1.ps.hmny.io"
+  ],
+  "faucets": ["http://dev.faucet.easynode.one/"],
+  "nativeCurrency": {
+    "name": "ONE",
+    "symbol": "ONE",
+    "decimals": 18
+  },
+  "infoURL": "https://www.harmony.one/",
+  "shortName": "hmy-ps-s0",
+  "chainId": 1666900000,
+  "networkId": 1666900000,
+  "explorers": [{
+    "name": "Harmony Block Explorer",
+    "url": "https://explorer.ps.hmny.io/",
+    "standard": "EIP3091"
+  }]
+}

**Comment by najmahmd** [comment]
diff --git a/_data/chains/eip155-1666900000.json b/_data/chains/eip155-1666900000.json
new file mode 100644
index 000000000..ce357349a
--- /dev/null
+++ b/_data/chains/eip155-1666900000.json
@@ -0,0 +1,23 @@
+{
+  "name": "Harmony Devnet Shard 0",
+  "chain": "Harmony",
+  "rpc": [
+    "https://api.s1.ps.hmny.io",
+    "https://api.s1.ps.hmny.io"
+  ],
+  "faucets": ["http://dev.faucet.easynode.one/"],
+  "nativeCurrency": {
+    "name": "ONE",
+    "symbol": "ONE",
+    "decimals": 18
+  },
+  "infoURL": "https://www.harmony.one/",
+  "shortName": "hmy-ps-s0",
+  "chainId": 1666900000,
+  "networkId": 1666900000,
+  "explorers": [{
+    "name": "Harmony Block Explorer",
+    "url": "https://explorer.ps.hmny.io/",
+    "standard": "EIP3091"
+  }]
+}

**Review by najmahmd — COMMENTED** [review]
_data/chains/eip155-1666900000.json

**Review by Marcin199626 — APPROVED** [review]

**Review by Marcin199626 — APPROVED** [review]

**Review by Apdlrcjafg19 — APPROVED** [review]
#
Duplicate of #_data/chains/eip155-1666900000.json

**Comment by SubhamKumarSadhukhan** [comment]
> 1666900000

**Comment by najmahmd** [comment]
gh pr checkout 1444

File: _data/chains/eip155-1666900000.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "symbol": "ONE",
+    "decimals": 18
+  },
+  "infoURL": "https://www.harmony.one/",
+  "shortName": "hmy-ps-s0",
+  "chainId": 1666900000,
+  "networkId": 1666900000,
+  "explorers": [{
+    "name": "Harmony Block Explorer",
+    "url": "https://explorer.ps.hmny.io/",
```
Comment (ligi) [inline _data/chains/eip155-1666900000.json:20]:
```suggestion
    "url": "https://explorer.ps.hmny.io",
```

**Review by ligi — APPROVED** [review]

**Pull request merged**